dojo/tools/bandit/parser.py

```diff
@@ -19,29 +17,31 @@ def get_description_for_scan_types(self, scan_type):
     def get_findings(self, filename, test):
         data = json.load(filename)
 
-        dupes = dict()
+        res = list()
```

Comment:
Can we have a more descriptive variable name here, eg. `results`?

Reply:
Fixed.